dispatch/static/manager/src/js/vendor/dispatch-editor/embeds/WidgetEmbed.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+import WidgetSelectInput from '../../../components/inputs/WidgetSelectInput'
+import { FormInput, TextInput } from '../../../components/inputs'
+
+function WidgetEmbedComponent(props) {
+  console.log(props)
+  return (
+    <div className='o-embed o-embed--widget'>
+
+        <FormInput label='Content'>
+
+          <TextInput
+            fill={true}
+            value={props.data.content}
+            onChange={e => props.updateField('content', e.target.value)} />
+
+        </FormInput>
+
+        <FormInput>
+          <WidgetSelectInput
+            zoneId={props.data.zoneId}
```

Comment:
- `zoneId` is always 'embed'
- the current widget ID should be stored in `data.widget_id`
```jsx
<WidgetSelectInput
  zoneId='embed' 
  selected={props.data.widget_id}
  update={widgetId => props.updateField('widget_id', widgetId)} />